.github/workflows/push-dist.yml

```diff
@@ -0,0 +1,21 @@
+# .github/workflows/push-dist.yml
+
+name: Push dist
+
+on:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  push-dist:
+    name: Push dist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: npm run build # Add whatever you need to build your pacakage here
```

Comment:
this is pretty close to what I'm using over here: https://github.com/NullVoxPopuli/ember-resources/blob/main/.github/workflows/push-dist.yml

but I think, because this repo is using yarn, we need these extra lines after the checkout:
https://github.com/emberjs/ember-test-helpers/blob/master/.github/workflows/ci-build.yml#L20-L21